index.html

```diff
@@ -164,6 +164,27 @@ <h3 class="skill">Skills</h3>
                 </div>
             </div>
         </section>
+        <footer class="contact">
+            <h2>I'm always <br> interested in hearing about new projects, 
+                so if you'd like to chat please get in</h2>
+            <form class="ip-type" action="https://formspree.io/f/xzbokgee">
```

Comment:
- [ ] Your not using a method for this form, kindly add a ```POST``` method to your form so it sends data to the form spree service